redirector.py

```diff
@@ -172,6 +171,22 @@ def get(self):
     def stats(self):
         return self.settings['stats']
 
+class APISpawnHandler(RequestHandler):
+    @gen.coroutine
+    def post(self):
+        random_host = select_host(self.stats)
+        http_client = AsyncHTTPClient()
+        request = HTTPRequest(random_host + "/api/spawn/", method="POST", body="")
+        response = yield http_client.fetch(request)
```

Comment:
We should check the response (status code) to make sure we really have a server to hand back to them.